### PR TITLE
Feat/prevent snapshot warnings

### PR DESCRIPTION
**Pull request by LarsDenBakker**
- prevent filename warnings from snapshot filenames
- don't warn about unused snapshots if you are using `grep` to run individual files

### CODE REVIEW

**Comment by LarsDenBakker** [comment]
@daKmoR I made sure to include all new files in the `package.json` this time :)

File: packages/testing-karma/default-config.js [hunk 1]
```diff
@@ -48,6 +48,7 @@ module.exports = config => ({
     require.resolve('karma-snapshot'),
     require.resolve('karma-mocha-snapshot'),
     require.resolve('karma-chrome-launcher'),
+    require.resolve(require.resolve('./src/snapshot-filename-preprocessor.js')),
```
Comment (daKmoR) [inline packages/testing-karma/default-config.js]: why is a nested `require.resolve(require.resolve(` needed?

Comment (LarsDenBakker) [reply packages/testing-karma/default-config.js]: It's not :D I didn't have my morning coffe when I wrote that, I think.

**Comment by LarsDenBakker** [comment]
merged into https://github.com/open-wc/open-wc/pull/387

**Pull request closed without merging**